scrabble.js

```diff
@@ -1,8 +1,72 @@
 var Scrabble = function() {};
 
-// YOUR CODE HERE
-Scrabble.prototype.helloWorld = function() {
-  return 'hello world!';
+// s = new Scrabble
+// s.helloWorld()
+
+Scrabble.helloWorld = function() {
+  return 'hello world!'
+};
+
+Scrabble.score = function(word) {
+
+  var upper_word = word.toUpperCase();
+
+  var score = 0
+
+  var mapObj = {
+    'A': 1, 'E': 1, 'I': 1, 'O': 1, 'U': 1, 'L': 1, 'N': 1, 'R': 1, 'S': 1, 'T': 1,
+    'D': 2, 'G': 2, 'B': 3, 'C': 3, 'M': 3, 'P': 3, 'F': 4, 'H': 4, 'V': 4, 'W': 4, 'Y': 4,
+    'K': 5, 'J': 8, 'X': 8, 'Q': 10, 'Z': 10
+  };
+
+  var bonus = 50;
+
```

Comment:
great job separating out these constants like bonus; in case they ever change, it'll be super easy for you to go in and update it